packages/desktop-client/src/components/reports/reports/CashFlow.tsx

```diff
@@ -52,7 +52,8 @@ function CashFlow() {
 
   useEffect(() => {
     async function run() {
-      const trans = await send('get-earliest-transaction');
+      let trans;
+      trans = await send('get-earliest-transaction');
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏previous implementation here was much better. What's the problem you're experiencing? There must be a different solution to it.

Reply:
I'm seeing "Property 'date' does not exist on type 'unknown'." on line 58 with 'const'. 'let' removed the error but not sure if there is a better way to do it

Reply:
You'll want to update [this line](https://github.com/actualbudget/actual/blob/b3bcff094db3d55445325357d943d1377181c808/packages/loot-core/src/types/server-handlers.d.ts#L57) to solve that problem. From `unknown` to a full-er datatype.